boards/common/nrf51/Makefile.features

```diff
@@ -0,0 +1,15 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_adc
+FEATURES_PROVIDED += periph_gpio periph_gpio_irq
+FEATURES_PROVIDED += periph_i2c
+FEATURES_PROVIDED += periph_rtt
+FEATURES_PROVIDED += periph_spi
+FEATURES_PROVIDED += periph_timer
+FEATURES_PROVIDED += periph_uart
```

Comment:
pulling all of these into the common folder seems to invalid to me, as there might be nrf51-based boards that do not provide all of these.

My suggestion: 
- all features that do not depend on board-specific (e.g. pin) configurations (gpio, gpio_irq) should be put into `cpu/nrf51/Makefile.features`. 
- features that are configured in the shared `boards/common/nrf51/include/periph_conf_common.h` should stay in this file
- all others should go into the corresponding board folders